ush/create_aqm_rc_file.py

```diff
@@ -113,35 +129,20 @@ def create_aqm_rc_file(cdate, run_dir, init_concentrations):
     #
     #-----------------------------------------------------------------------
     #
-    try:
-        fill_jinja_template(
+    with tempfile.NamedTemporaryFile(dir="./", mode="w+t", prefix="aqm_rc_settings") as tmpfile:
```

Comment:
@christinaholtNOAA, I got the following error in 'run_fcst':
```
Traceback (most recent call last):
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/create_aqm_rc_file.py", line 179, in <module>
    create_aqm_rc_file(
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/create_aqm_rc_file.py", line 13
5, in create_aqm_rc_file
    set_template(
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/python_utils/workflow-tools/scripts/templater.py", line 105, in set_template
    user_args = parse_args(argv)
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/python_utils/workflow-tools/scripts/templater.py", line 79, in parse_args
    return parser.parse_args(argv)
  File "/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/lib/python3.8/argparse.py", 
line 1768, in parse_args    args, argv = self.parse_known_args(args, namespace)
  File "/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/lib/python3.8/argparse.py", 
line 1800, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)  File "/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/lib/python3.8/argparse.py", 
line 1841, in _parse_known_args    option_tuple = self._parse_optional(arg_string)
  File "/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/lib/python3.8/argparse.py", 
line 2128, in _parse_optional
     if not arg_string[0] in self.prefix_chars:
TypeError: '_TemporaryFileWrapper' object is not subscriptable
+ exregional_run_fcst.sh[587]: print_err_msg_exit 'Call to function to create an aqm.rc file for the current
cycle'\''s (cdate) run directory (DATA) failed:
```

Reply:
@christinaholtNOAA, I got another error in AQM:
```
Traceback (most recent call last):
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/create_aqm_rc_file.py", line 179
, in <module>
    create_aqm_rc_file(
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/create_aqm_rc_file.py", line 135
, in create_aqm_rc_file
    set_template(
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/python_utils/workflow-tools/scripts/templater.py", line 119, in set_template
    cfg = setup_config_obj(user_args, log_name=log.name)
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/python_utils/workflow-tools/scri
pts/templater.py", line 88, in setup_config_obj    config_type = cli_helpers.get_file_type(user_args.config_file)
  File "/lfs/h2/emc/lam/noscrub/chan-hoo.jeon/srw_pr_test/ufs-srweather-app/ush/python_utils/workflow-tools/src/uwtools/utils/cli_helpers.py", line 33, in get_file_type
    raise ValueError(msg)
ValueError: Bad file suffix -- . Cannot determine file type!
```
You can check this error on Hera too. The sample script of the community mode of AQM is available on Hera.

Reply:
I'm sorry! I did miss this one! I will tackle that now.